### DISCUSSION TITLE
NodeMCU and Smart Meter gets no Data from the electricity meter

### DISCUSSION THREAD

**Discussion by TiBros** [arendst/Tasmota#17146] [Q&A]
Flashed a NodeMCU with Tasmota and the SML extensions. I installed a bitShake Smart Meter Reader (all lines checked OK) and attached it to my Easymeter Q3A. Can see the Tasmota interface but can't get any data. Of course, I entered the pin on the electricity meter and checked the IR diodes using a cell phone camera. What can I do?

**Comment by mrRobot62** [comment]
Not sure if you can work with the same configuration but I'm using a Q3D SmartMeter
Hope that helps
```
D
B
=>sensor53 r
M 1
+1,3,o,0,9600,Q3D,1,10,2F3F210D0A
1,1-0:1.8.0255(@1,E_in,kWh,E_in,19
1,1-0:21.7.0255(@1,L1,W,L1,18
1,1-0:41.7.0255(@1,L2,W,L2,18
1,1-0:61.7.0255(@1,L3,W,L3,18
1,1-0:1.7.0255(@1,P_in,W,P_in,18
1,1-0:0.0.0255(@1,Netzbetreiber-ID,,NetID,0
1,0-0:96.1.255*255(@#),Seriennummer,,serial,0
```
inside console you can test if configuration works enter
`sensor53 d1`
please note nothing will be transferred via MQTT
Please check main-menu, you should see above the menu some entries from your config
![image](https://user-images.githubusercontent.com/949032/203159100-083e5a38-1735-4b25-9cee-802e80b5a7cb.png)

use 
`sensor53 d0`
to close dump mode

Some more infos
E_in, L1, L2, L3, P_in, NetID  => MQTT topics you will found the data as values 
+1,3..... = Device 1 and GPIO3 (default GPIO, please check this double ;-)
o = OBIS ASCII
0,9600 = 0=counter with pull-up, 9600=Baud


Please check this wiki entry
https://tasmota.github.io/docs/Smart-Meter-Interface/#easymeter-q3a-apator-apox-sml

**Reply by TiBros** [reply]
Thx. I´ll try it this afternoon.
I'll let you know.

**Reply by TiBros** [reply]
unfortunately the same result. no data!! but the q3a is also a 2-Tariff Meter.
